### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by cjt1105**

### CODE REVIEW

**Comment by cjt1105** [comment]
@morecallan

File: word-search.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+"use strict"
+
+const fs = require('fs'),
+es = require('event-stream'),
+[,,args] = process.argv,
+stream = fs.createReadStream('/usr/share/dict/words','UTF8'),
+tStream = require('./limit-ten').tStream;
+let data = "",
```
Comment (morecallan) [inline word-search.js]: Don't need this variable anymore.

File: word-search.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+count = null;
```
Comment (morecallan) [inline word-search.js]: Same here.

**Comment by morecallan** [comment]
@cjt1105 Almost there! Just need to add comments to clarify your code. Also, try to get more specific with your variable/function names then you are ready to merge.

**Pull request merged**